### PR TITLE
fix: db files

### PR DESCRIPTION
**Pull request by lennartkloock**
## Proposed changes

Some small fixes for db files.

## Types of changes

What types of changes does your code introduce to 7TV?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/SevenTV/SevenTV/blob/main/CONTRIBUTING.md) doc
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

## Further comments

### CODE REVIEW

File: database/emotes.sql [hunk 1]
```diff
@@ -13,6 +13,12 @@ CREATE TABLE "emotes" (
 CREATE INDEX "emotes_owner_id_index" ON "emotes" ("owner_id");
 CREATE UNIQUE INDEX "emotes_ticket_id_unique" ON "emotes" ("ticket_id");
 
+CREATE TABLE "emote_attributions" (
+    "emote_id" uuid NOT NULL, -- Ref: emotes.id -> On Delete Cascade
+    "user_id" uuid NOT NULL, -- Ref: users.id -> On Delete Cascade
+    PRIMARY KEY ("emote_id", "user_id")
```
Comment (TroyKomodo) [inline database/emotes.sql:19]: index must be on user_id too

File: database/emotes.sql [hunk 2]
```diff
@@ -24,6 +30,7 @@ CREATE INDEX "emote_files_file_id_index" ON "emote_files" ("file_id");
 CREATE TABLE "emote_sets" (
     "id" uuid PRIMARY KEY,
     "owner_id" uuid, -- Ref: users.id -> On Delete Cascade
+    "hidden_on_profile" bool NOT NULL DEFAULT FALSE,
```
Comment (TroyKomodo) [inline database/emotes.sql:33]: perhaps this could be a flag 🤔

File: database/users.sql [hunk 3]
```diff
@@ -35,7 +35,8 @@ CREATE INDEX "users_entitled_cache_invalidate_product_ids_index" ON "users" USIN
 CREATE INDEX "users_active_badge_id_index" ON "users" ("active_badge_id");
 CREATE INDEX "users_active_paint_id_index" ON "users" ("active_paint_id");
 CREATE INDEX "users_active_profile_picture_id_index" ON "users" ("active_profile_picture_id");
-CREATE INDEX "users_active_connection_id_index" ON "users" ("active_connection_id");
+
+CREATE TYPE "connection_platform" AS ENUM ('DISCORD', 'TWITCH', 'YOUTUBE');
```
Comment (TroyKomodo) [inline database/users.sql:39]: KICK aswell

File: database/emotes.sql (hunk 2 above)
Comment (lennartkloock) [reply database/emotes.sql:33]: not sure, I thought flags are meant to be public flags not related to the owner

Comment (TroyKomodo) [reply database/emotes.sql:35]: they can be anything

**Pull request merged**